src/scss/dataTable.scss

```diff
@@ -84,6 +84,16 @@
       word-wrap: break-word;
     }
 
+    td {
+      i {
+        display: none;
+      }
+
+      &:hover i {
```

Comment:
We need the icon to be visible if user hovers over the row, not the cell.

Reply:
@tonytlwu Fixed.
New [screencast](https://share.getcloudapp.com/9ZuXGEOm)